index.js

```diff
@@ -103,8 +103,8 @@ module.exports = {
       'components/dialog/dialog-theme.scss'
     ];
 
-    var pathBase = this.project.addonPackages['ember-paper'].path;
-    var angularMaterialPath =  'node_modules/angular-material-source/src';
+    var pathBase = this.project.nodeModulesPath;
```

Comment:
sure - prior to this change it was looking for angular-material-source, from my ember-cli terminal(for my project) at `node_modules/ember-paper/node_modules/angular-material-source/src`. This works fine for the dummy app b/c the lookup path then becomes `node_modules/angular-material-source/src`. To make this work when installing it in another app however, I changed it to be the `node_modules/angular-material-source` by using the current projects node_modules folder as the start of the lookup path instead of ember-paper's root dir only.

as an analogy - the net affect is like using process.cwd() instead of __dirname when making a node module - the lookup path is now relative the consumers root path and the source modules root too.